shoebot/util/fontnames.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+import gi
+gi.require_version('PangoCairo', '1.0')
+from gi.repository import PangoCairo
+# https://developer.gnome.org/pygtk/stable/index.html
+
+def fontnames():
+    fontstrings = []
+    families = PangoCairo.font_map_get_default().list_families()
+
+    for family in families:
+        for face in family.list_faces():
+            # FIXME: Ensure this is the correct way to make a font name
+            fontstrings.append(family.get_name() + ' ' + face.get_face_name())
+            # description = face.describe()
```

Comment:
Nothing blocking, just some stylistic nits (mostly more modern python):

- Since it's a fontname (but fontname is a shoebot reserved word) maybe a name like `names` would be more descriptive (it's a bit redundant to have string as part of the name)
- for building the font name an fstring is more readable these days 
- maybe consider doing everything in a list comprehension
- consider a docstring or comment with a real example of a Pango string and what the output is, as without knowing about Pango and font formats it's impossible to judge what this will return (this is the kind of info we need for tests too)

Apologies for so much feedback, this is as actually a good little function, and nothing really big in the feedback :)

Reply:
Thanks so much, this is all relevant! Addressed in ba0cd6b9a0a71caed3e852e63b7c9a4f9dfad20a